libphobos/libdruntime/core/cpuid.d

```diff
@@ -676,58 +676,61 @@ void getCpuInfo0B()
         }
         ++level;
     } while (a!=0 || b!=0);
-
 }
 
 void cpuidX86()
 {
     auto cf = getCpuFeatures();
 
-    char * venptr = cf.vendorID.ptr;
-    uint a, b, c, d, a2;
+    uint a, b, c, d;
+    uint* venptr = cast(uint*)cf.vendorID.ptr;
```

Comment:
core.cpuid is now identical to upstream.  Probably a bit belated, I've left it in a half patched version for too long (sorry @mleise!)